reference/api/multi_search.md

```diff
@@ -0,0 +1,122 @@
+# Multi-index search
```

Comment:
```suggestion
# Multi-index search

Often called federated search, it is a technique for searching indexes simultaneously and then presenting the results in a unified interface.
```

@maryamsulemani97 Added the term federated search here with a quick explanation

Reply:
@guimachiavelli It was a wrong statement from the initial squad! 

Federated-search means you search across differences sources (like multi-index). Aggregating the results or having them separated is a detail (there is no direct naming for the different ways)

Reply:
Got it, thanks for the explanation ☺️